packages/go/test/fixtures/20-square-brackets/vercel.json

```diff
@@ -1,10 +1,13 @@
 {
   "version": 2,
-  "builds": [{ "src": "api/**/*.go", "use": "@vercel/go" }],
```

Comment:
We need to remove this so that `zeroConfig` can pick this up and the dynamic segment behavior of bracket routes can be tested.